Mathlib/Algebra/Module/LocalizedModule.lean

```diff
@@ -1146,6 +1171,13 @@ theorem map_exact (g : M₀ →ₗ[R] M₁) (h : M₁ →ₗ[R] M₂) (ex : Func
   Function.Exact.of_ladder_linearEquiv_of_exact
     (map_iso_commute S f₀ f₁ g) (map_iso_commute S f₁ f₂ h) (LocalizedModule.map_exact S g h ex)
 
+/-- Localization of composition is the composition of localization -/
+theorem map_comp' (g : M₀ →ₗ[R] M₁) (h : M₁ →ₗ[R] M₂) :
+    map S f₀ f₂ (h ∘ₗ g) = map S f₁ f₂ h ∘ₗ map S f₀ f₁ g := by
```

Comment:
Should this be called `map_comp_map` as in `IsLocalization.map_comp_map`? Also `IsLocalization.map_comp_map` is stated the other way round, but your version looks like the correct order. So I guess we keep this order and remember to fix `IsLocalization.map_comp_map` one day?

Reply:
Oh that makes sense. Then I guess it is `map_comp` that should be named `map_comp_algebraMap` instead, but not in this PR.